build.gradle

```diff
@@ -18,7 +18,7 @@ if (ciBuild) {
 
 jenkinsPlugin {
   // Version of Jenkins core this plugin depends on.
-  coreVersion = '1.642.1'
+  coreVersion = '2.60.3'
```

Comment:
Can we get around rising the base version by adding a new test task which tests against a different Jenkins version? Can you try to make this happen?

I nearly prefer having a unit test which tests for the specific problem and not incrementing the base verison.

Reply:
I added a "jcascTest" configuration + source set + task to make it happen. The commit comment describes all the details.

I am not 100% satisfied. The IDEA integration cannot handle this setup properly. But that's the best I can come up with, given my current Gradle knowledge and available time.